INSTALL.rst

```diff
@@ -166,22 +146,9 @@ the only one that will be saved across different container runs.
 Platform Support
 ^^^^^^^^^^^^^^^^
 
-Users of Mac M1 or M2 chips and Windows users may have issues running certain components of CKT.
-
-If you are using Linux or macOS with an Intel chip (i.e., not the
-new M1 or M2 chips), everything should work natively, so we
-recommend either :ref:`Option 1` or :ref:`Option 2`.
-
-All users on ARM chips, as well as all Windows users, may have to
-take care when installing the toolbox, depending on which tools they
-intend to use.
-  
-  - The automatic wire cut search in the ``cutqc`` package depends
-    on CPLEX, which is only available on Intel chips and is not yet available
-    for Python 3.12.
-
-In each case, one method that is guaranteed to work is to :ref:`use
-the toolbox within Docker <Option 3>`.  Other methods include:
+We expect this package to work on `any platform supported by Qiskit <https://docs.quantum.ibm.com/start/install#operating-system-support>`__. If
+you are experiencing issues running the software on your device, you
+may consider :ref:`using the toolbox within Docker <Option 3>`.
```

Comment:
IMO the file should end with this line. We no longer need the bullet point after this -- about installing an x86 version of python or using conda -- now that pyscf and cplex are both no longer required.